FFBardMusicPlayer/DryWetUtil.cs

```diff
@@ -245,10 +245,12 @@ public static MemoryStream ScrubFile(string filePath)
 
                     int octaveShift = 0;
                     string trackName = originalChunk.Events.OfType<SequenceTrackNameEvent>().FirstOrDefault()?.Text;
-                   
+
+                    if (trackName == null)
+                        trackName = "";
```

Comment:
This breaks null safety, instead look at https://github.com/BardMusicPlayer/BardMusicPlayer/commit/7a21949389f699f514af60e0719f0a611caa2687 as this might have adverse effects that is hard to track down later down the line.

Reply:
I didn't even notice it was done a few lines after T_T